src/Game/UI/Gumps/WorldMapGump.cs

```diff
@@ -225,7 +228,13 @@ private void BuildOptionDictionary()
             _options["flip_map"] = new ContextMenuItemEntry("Flip map", () => { _flipMap = !_flipMap; }, true, _flipMap);
             _options["top_most"] = new ContextMenuItemEntry("TopMost", () => { TopMost = !TopMost; }, true, _isTopMost);
             _options["free_view"] = new ContextMenuItemEntry("Free view", () => { FreeView = !FreeView; }, true, FreeView);
-            _options["show_party_members"] = new ContextMenuItemEntry("Show party members", () => { _showPartyMembers = !_showPartyMembers; }, true, _showPartyMembers);
+            _options["show_party_members"] = new ContextMenuItemEntry("Show party members", () =>
+            {
+                _showPartyMembers = !_showPartyMembers;
+
+                World.WMapManager.SetEnable(_showPartyMembers);
```

Comment:
I think it's a good idea to add a timers reset. Just `_lastSOMETHINGTIMER = 0` or you will see a delay when you will re-enable the option

Reply:
Using `World.WMapManager.SetEnable` will trigger it immediately based on my testing.  No delay when enabling/disabling it.

![2020-03-25_11-39-08](https://user-images.githubusercontent.com/577652/77561849-411bb380-6e8d-11ea-9245-073e55d8e9bc.gif)